lib/jekyll/collection.rb

```diff
@@ -32,7 +32,8 @@ def respond_to?(method, include_private = false)
     # Override of method_missing to check in @data for the key.
     def method_missing(method, *args, &blck)
       if docs.respond_to?(method.to_sym)
-        Jekyll.logger.warn "Deprecation:", "#{label}.#{method} should be changed to #{label}.docs.#{method}."
+        Jekyll.logger.warn "Deprecation:", "#{label}.#{method} should be changed to
+          #{label}.docs.#{method}."
```

Comment:
```
"String1" \
  "string2"
```

What you have there will be almost literally output.  Ruby has the concept of split strings so you can do:

```
"hello " \
  "world"
```

And you'll get "hello world" instead of "hello\n      world"